packages/flutter/test/material/date_picker_test.dart

```diff
@@ -774,3 +777,70 @@ void _tests() {
     expect(tester.getBottomLeft(find.text('OK')).dx, 800 - ltrOkRight);
   });
 }
+
+void _screenConfigTests() {
+
+  const Size kPixel1Portrait = Size(1070, 1770);
```

Comment:
A comment about these sizes, like what they're based on, why we chose them, would be helpful.

Reply:
Just pushed a fix that renamed them and commented them with hopefully better descriptions.